packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss

```diff
@@ -221,6 +222,18 @@
         }
     }
 
+    &-Price {
+        flex: 1;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-end;
+        text-align: end;
+
+        strong {
```

Comment:
It is better to specify class for the item you want to style instead of applying styling to all `strong` tags.